styles/content.css

```diff
@@ -185,17 +185,11 @@
   @apply text-2xl;
```

Comment:
I had added these during the work to add the test-type banner in various locations. It was addressing specific content that was wrapping and is no longer relevant. Worse, the byproduct was borking in other places. I reviewed the list we created for that audit and looked through these as a confirmation that we're 👍